clustered/osgi-test/build.gradle

```diff
@@ -91,6 +91,14 @@ task unzipKit(type: Copy) {
 test {
   dependsOn unzipKit
   systemProperty 'kitInstallationPath', "$unzipKit.destinationDir/${project(':clustered:clustered-dist').archivesBaseName}-$project.version-kit"
+
+  // do not cache test execution
+  outputs.cacheIf { false }
+  // adds the ability, like Maven, to run only unit tests but skip IT/system tests
+  if(project.hasProperty('skipITs')) {
```

Comment:
If we're going to do this I'd rather tie the kit tests to a distinct task instance `integrationTest` for example. Then `check.dependsOn integrationTest` and `./gradlew test` only runs unit tests and `integrationTest` or `check` will run the kit tests as well. That said here are the current timings for various executions on my dev machine:
```
./gradlew cleanTest test
...
BUILD SUCCESSFUL in 4m 30s
143 actionable tasks: 36 executed, 107 up-to-date

./gradlew cleanTest test -x :clustered:integration-test:test -x :clustered:osgi-test:test
...
BUILD SUCCESSFUL in 2m 34s
122 actionable tasks: 33 executed, 89 up-to-date
```
Assuming things aren't regressing too badly with this PR then I'm not sure any change here is really needed.

Reply:
I will remove from the PR when it will be ready. Don't want to loose time with that.
I just wanted a way to first run the unit tests without the system tests. It helps having such flags in tc-platform and EE because systems tests takes more than 1h in these projects.

Reply:
removed